examples/ts/fetch-orderbook.ts

```diff
@@ -1,12 +1,14 @@
-import { binance } from '../../ts/ccxt.js'
```

Comment:
@ttodua I was doing this specific import on purpose to showcase that we don't need to import the entire package to use only one or two exchanges

Reply:
ah ok.
which way do you suggest, so it remained transpilable.

Reply:
yes, we can have both examples inside same file, maybe we shouldnt create separate files just to teach users how to import. we can smth like this in same file:

```
import { binance } from '../../ts/ccxt.js'; // preferred approach
import ccxt from '../../ts/ccxt.js';
...
b1 = new binance(); // preferred approach
b2 = new ccxt['binance'];
```
however, I think we will need some changes in transpiler (or maybe in AST-Transpiler)?

Reply:
@ttodua Here the goal is only to show the import so the name can express that `import-one-exchange-esm-example` for instance.